src/components/common/Button.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+const styles = {
+  buttonStyles:{
+    fontSize: '1em',
+    padding: '2%',
+    borderRadius: '100em',
+    background: '#f8f8f8',
+    color: '#333'
+  },
+}
+
+const Button = props => (
+  <button style={styles.buttonStyles} onClick={props.onClick} {...props}>{props.children}</button>
```

Comment:
`props.onClick` will be unpacked with the other props in `{ ...props }`

Reply:
Love it.